### PR TITLE
Add Turkish Language File

### PR DESCRIPTION
**Pull request by adilarli**

### CODE REVIEW

**Comment by stianst** [comment]
Thanks, we need to find someone from the community that can review this

**Comment by stianst** [comment]
@keycloak-ci-bot test

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+accountManagementWelcomeMessage=Keycloak Hesap Y\u00F6netimine Ho\u015F Geldiniz
+personalInfoIntroMessage=Temel bilgilerinizi y\u00F6netin
+accountSecurityTitle=Hesap G\u00FCvenli\u011Fi
+accountSecurityIntroMessage=\u015Eifrenizi ve hesap eri\u015Fiminizi kontrol edin
+applicationsIntroMessage=Hesab\u0131n\u0131za eri\u015Fmek i\u00E7in uygulama izninizi takip edin ve y\u00F6netin
+resourceIntroMessage=Kaynaklar\u0131n\u0131z\u0131 ekip \u00FCyeleri aras\u0131nda payla\u015F\u0131n
+passwordLastUpdateMessage=\u015Eifreniz g\u00FCncellendi
+updatePasswordTitle=\u015Eifre g\u00FCncelle
+updatePasswordMessageTitle=G\u00FC\u00E7l\u00FC bir \u015Fifre se\u00E7ti\u011Finizden emin olun
+updatePasswordMessage=G\u00FC\u00E7l\u00FC bir \u015Fifre, say\u0131lar, harfler ve sembollerin bir kar\u0131\u015F\u0131m\u0131n\u0131 i\u00E7erir. Tahmin etmek zor, ger\u00E7ek bir kelimeye benzemeyen ve sadece bu hesap i\u00E7in kullan\u0131l\u0131r.
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: updatePasswordMessage=G\u00FC\u00E7l\u00FC bir \u015Fifre, say\u0131lar, harfler ve sembollerin kar\u0131\u015F\u0131m\u0131ndan olu\u015Fmal\u0131d\u0131r. Tahmin etmesi zor ve ger\u00E7ek bir kelimeye benzemeyen \u015Fifre sadece bu hesap i\u00E7in kullan\u0131l\u0131r.

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 2]
```diff
@@ -0,0 +1,350 @@
...
+password=\u015Eifre
+currentPassword=\u015Eimdiki \u015Eifre
+passwordConfirm=\u015Eifre Do\u011Frulama
+passwordNew=Yeni \u015Eifre
+username=Kullan\u0131c\u0131 Ad\u0131
+address=Adres
+street=Cadde
+region=B\u00F6lge
+postal_code=Posta Kodu
+locality=Semt
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: locality = \u015Eehir

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 3]
```diff
@@ -0,0 +1,350 @@
...
+backToApplication=&laquo; Uygulamaya D\u00F6n
+backTo=Geri D\u00F6n {0}
+
+date=G\u00FCn
+event=Olay
+ip=IP
+client=\u0130stemci
+clients=\u0130stemciler
+details=Detaylar
+started=Ba\u015Flang\u0131\u00E7 Tarihi
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: If 'Tarihi' is used here (which means the **date**) is should also used for the next 2 lines.

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 4]
```diff
@@ -0,0 +1,350 @@
...
+lastAccess=Son Eri\u015Fim
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: lastAccess=Son Eri\u015Fim Tarihi

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 5]
```diff
@@ -0,0 +1,350 @@
...
+expires=Son Kullanma
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: expires=Son Kullanma Tarihi

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 6]
```diff
@@ -0,0 +1,350 @@
...
+federatedIdentity=Federal Kimlik
+authenticator=Kimlik Do\u011Frulama
+device-activity=Cihaz Etkinli\u011Fi
+sessions=Oturum
+log=Log
+
+application=Uygulama
+availablePermissions=Kullan\u0131labilir \u0130zinler
+availableRoles=Kullan\u0131labilir Roller
+grantedPermissions=\u0130zin Verilen \u0130zinler
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: grantedPermissions= Verilen \u0130zinler

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 7]
```diff
@@ -0,0 +1,350 @@
...
+resourceNoPermissionsGrantingAccess=Bu kayna\u011Fa eri\u015Fim izni verilmeyen izin yok
+anyAction=Herhangi bir eylem
+description=A\u00E7\u0131klama
+name=\u0130sim
+scopes=Kapsam
+resource=Kaynak
+user=Kullan\u0131c\u0131
+peopleSharingThisResource=Bu kayna\u011F\u0131 payla\u015Fan kullan\u0131c\u0131lar
+shareWithOthers=Ba\u015Fkalar\u0131yla payla\u015F
+needMyApproval=Onay\u0131ma ihtiyac\u0131m var
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: needMyApproval=Onay\u0131m gerekli

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 8]
```diff
@@ -0,0 +1,350 @@
...
+requestsWaitingApproval=Talepleriniz onay bekliyor
+icon=Icon
+requestor=Talep eden
+owner=Sahip
+resourcesSharedWithMe=Kaynaklar benimle payla\u015F\u0131ld\u0131
+permissionRequestion=\u0130zin \u0130stekleri
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: permissionRequestion=\u0130zin Talepleri

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 9]
```diff
@@ -0,0 +1,350 @@
...
+permission=\u0130zin
+shares=payla\u015F\u0131m(lar)
+
+locale_ca=Katalanca
+locale_de=Almanca
+locale_en=\u0130ngilizce
+locale_es=\u0130spanyolca
+locale_fr=Frans\u0131zca
+locale_it=\u0130talyanca
+locale_ja=Afrikanca
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: not sure about this, is it 	Japanese (Japonca)?
locale_ja=Japonca

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 10]
```diff
@@ -0,0 +1,350 @@
...
+permissionRequests=\u0130zin \u0130stekleri
+approve=Onayla
+approveAll=T\u00FCm\u00FCn\u00FC onayla
+sharedwith=\u0130le payla\u015Ft\u0131
+people=insanlar
+perPage=Sayfa ba\u015F\u0131na
+currentPage=Ge\u00E7erli sayfa
+sharetheResource=Kayna\u011F\u0131 payla\u015F
+user=Kullan\u0131c\u0131
+group=Gurup
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: group=Grup

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 11]
```diff
@@ -0,0 +1,350 @@
...
+
+role_admin=Admin
+role_realm-admin=Realm Admin
+role_create-realm=Realm Olu\u015Ftur
+role_view-realm=Realm g\u00F6r\u00FCnt\u00FCle
+role_view-users=Kullan\u0131c\u0131lar\u0131 g\u00F6r\u00FCnt\u00FCle
+role_view-applications=Uygulamalar\u0131 g\u00F6r\u00FCnt\u00FCle
+role_view-clients=\u0130stemci g\u00F6r\u00FCnt\u00FCle
+role_view-events=Olay g\u00F6r\u00FCnt\u00FCle
+role_view-identity-providers=Identity Provider g\u00F6r\u00FCnt\u00FCle
```
Comment (AlicanAkkus) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: should be 'Kimlik Sağlayıcılar'

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 12]
```diff
@@ -0,0 +1,350 @@
...
+role_manage-realm=Realm y\u00F6net
+role_manage-users=Kullan\u0131c\u0131lar\u0131 y\u00F6net
+role_manage-applications=Uygulamalar\u0131 y\u00F6net
+role_manage-identity-providers=Identity Provider y\u00F6net
```
Comment (AlicanAkkus) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: should be 'Kimlik Sağlayıcıları Yönet'

**Review by batuhankok — DISMISSED** [review]

**Comment by stianst** [comment]
@AlicanAkkus @neslihanedes can you review again and approve if you are happy with changes?

**Review by AlicanAkkus — DISMISSED** [review]

**Comment by AlicanAkkus** [comment]
> @AlicanAkkus @neslihanedes can you review again and approve if you are happy with changes?

looks like good. 👍🏻

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 13]
```diff
@@ -0,0 +1,350 @@
...
+log=Log
+
+application=Uygulama
+availablePermissions=Kullan\u0131labilir \u0130zinler
+availableRoles=Kullan\u0131labilir Roller
+grantedPermissions=Verilen \u0130zinler
+grantedPersonalInfo=\u0130zin Verilen Ki\u015Fisel Bilgiler
+additionalGrants=Ek \u0130zinler
+action=Aksiyon
+inResource=kaynak
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: `inResource=Kaynak`

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 14]
```diff
@@ -0,0 +1,350 @@
...
+shareWithOthers=Ba\u015Fkalar\u0131yla payla\u015F
+needMyApproval=Onay\u0131m gerekli
+requestsWaitingApproval=Talepleriniz onay bekliyor
+icon=Icon
+requestor=Talep eden
+owner=Sahip
+resourcesSharedWithMe=Kaynaklar benimle payla\u015F\u0131ld\u0131
+permissionRequestion=\u0130zin Talepleri
+permission=\u0130zin
+shares=payla\u015F\u0131m(lar)
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: `shares=Payla\u015F\u0131m(lar)`

File: themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties [hunk 15]
```diff
@@ -0,0 +1,350 @@
...
+application=Uygulama
+date=Tarih
+sharedwith=\u0130le payla\u015Ft\u0131
+owner=Sahip
+accessPermissions=Eri\u015Fim \u0130zinleri
+permissionRequests=\u0130zin \u0130stekleri
+approve=Onayla
+approveAll=T\u00FCm\u00FCn\u00FC onayla
+sharedwith=\u0130le payla\u015Ft\u0131
+people=insanlar
```
Comment (neslihanedes) [inline themes/src/main/resources-community/theme/base/account/messages/messages_tr.properties]: `people=\u0130nsanlar`

**Review by neslihanedes — CHANGES_REQUESTED** [review]
I have suggested some capital letter corrections, the rest seems fine :)

**Comment by stianst** [comment]
@AlicanAkkus thanks

@neslihanedes thanks for second review,. @adilarli has pussed changes for capital letters, can you let me know if you are happy with the PR now?

**Review by neslihanedes — APPROVED** [review]

**Review by neslihanedes — APPROVED** [review]

**Comment by neslihanedes** [comment]
@stianst I'm happy with the last changes :)

**Pull request merged**